Support fully qualified paths to plugins

christian-bromann opened the pull request:
## Proposed changes

closes #4867

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)


### Reviewers: @webdriverio/project-committers

mgrybyk commented on packages/wdio-utils/src/initialisePlugin.js:
```diff
@@ -8,9 +9,9 @@ import { safeRequire } from './utils'
+     * directly import packages that are scoped or start with an absolute path
      */
-    if (name[0] === '@') {
+    if (name[0] === '@' || name[0] === path.sep) {
```
[path.sep](https://nodejs.org/api/path.html#path_path_sep) is either `/` or `\`.

How to handle the following cases:
- `./foo/bar` mentioned in the doc
- `c:\foo\bar`
- or just `foo\bar`

christian-bromann commented:
@mgrybyk I replaced the check with `path.isAbsolute` which should work for Linux and Windows systems. Also updated the docs to be implicit about using absolute paths.
